Dynamic section reports

nericksen opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add dynamic section scripts for generating indicator and incident data summaries

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

bakatzir commented on Packs/DynamicSectionReports/README.md:
```diff
@@ -0,0 +1,9 @@
+Dynamic sections allow you to populate content into indicator and incident layouts using an automation script.
+The `DisplayIndicatorReputationContent` automation contained in this pack makes it easy to display all reputation information collected for a particular indicator directly on its layout.
+
+<img src="https://github.com/demisto/content/raw/0ab37a70cd964e903a8fae02501bff3577c1fc67/Packs/DynamicSectionReports/doc_files/indicator_report.png" width="500px" />
```
can you share a print screen? I suggest using relative paths, https://xsoar.pan.dev/docs/documentation/readme_file#relative-image-urls
but your call.

nericksen replied:
will take a look and update, thanks!

nericksen replied:
Ahh I did try to use relative links to start but I received error in the linter

```
  Packs/DynamicSectionReports/README.md: [RM108] - Error in readme image:
Detected the following image relative path: (doc_files/indicator_report.png).
Relative paths are not supported in pack README files. See https://xsoar.pan.dev/docs/integrations/integration-docs#images for further info on how to add images to pack README files.
```

bakatzir replied:
sorry, it is supported only for non pack readme files.

bakatzir commented on Packs/DynamicSectionReports/Scripts/DisplayIndicatorReputationContent/DisplayIndicatorReputationContent.yml:
```diff
@@ -0,0 +1,16 @@
+commonfields:
+  id: 6acdd9d4-b42d-4303-8d72-bd3912696257
```
```suggestion
  id: DisplayIndicatorReputationContent
```

bakatzir commented on Packs/DynamicSectionReports/Scripts/DisplayTaggedWarroomEntries/DisplayTaggedWarroomEntries.yml:
```diff
@@ -0,0 +1,20 @@
+  description: 'The JSON filter for warroom entries described here: https://xsoar.pan.dev/docs/integrations/get-entries-filter'
+  name: filter
+commonfields:
+  id: e006e0b8-acfb-4725-8ff5-03d29152f6fb
```
```suggestion
  id: DisplayTaggedWarroomEntries
```

bakatzir reviewed (commented):
good thought using the script tags.
please consider generating readmes for the scripts. this is only a recommendation.

nericksen commented:
thanks for the feedback @bakatzir will update today with the suggested changes.

nericksen commented:
Okay I generated the script READMEs and added some more screenshots to the main README.  Also corrected the lint errors and script IDs. Let me know if there is anything else required.
